wam2layers/preprocessing/preprocess_era5_modellevels.py

```diff
@@ -98,29 +101,28 @@ def load_data(variable, date, levels=None):
     else:
         # fluxes will be calculated based on the column water vapour
         cw = cwv
+    cw = cw.assign_attrs(units="kg m-2")
 
     # Integrate fluxes and states to upper and lower layer
     lower_layer = dp_modellevels.lev > boundary
     upper_layer = ~lower_layer
 
     # Vertically integrate state over two layers
-    s_lower = cw.where(lower_layer).sum(dim="lev")
-    s_upper = cw.where(upper_layer).sum(dim="lev")
+    s_lower = cw.where(lower_layer).sum(dim="lev", keep_attrs=True)
+    s_upper = cw.where(upper_layer).sum(dim="lev", keep_attrs=True)
 
     # Determine the fluxes
-    fx = u * cw  # eastward atmospheric moisture flux (kg m-1 s-1)
-    fy = v * cw  # northward atmospheric moisture flux (kg m-1 s-1)
+    fx = (u * cw).assign_attrs(units="kg m-1 s-1")  # eastward atmospheric moisture flux (kg m-1 s-1)
+    fy = (v * cw).assign_attrs(units="kg m-1 s-1")  # northward atmospheric moisture flux (kg m-1 s-1)
 
```

Comment:
Is it not a bit tricky to assign units? In case the units are not correct? But when you do the check before it should be fine

Reply:
It is. Unfortunately, xarray doesn't check units when doing maths. It happily let's you add metres to kilograms. There is a similar package, iris, that is much stricter on this. However, in my experience, it is quite a bit more difficult to work with.

By adding this check and adding units in general, I hope that users will be reminded of the fact that they need to be careful. However, it remains their own responsibility.

Reply:
When you do the check before, it fails, because there are no units at all...